### DISCUSSION TITLE
Question : What's the story for System.Drawing?

### DISCUSSION THREAD

**Discussion by WebAssemblyMan** [dotnet/maui#112] [General]
Will MAUI have System.Drawing? 

We have an UWP application that requires System.Drawing. Moving to MAUI, should we plan for native drawing for each platform, evaluate library such as SkiaSharp, or assume we will get some kind of standardize System.Maui.Drawing library? Thanks.

**Comment by Happypig375** [comment]
[Xamarin.Forms will have drawing implemented](https://github.com/xamarin/Xamarin.Forms/issues/2452), so MAUI will get it too.

**Comment by WebAssemblyMan** [comment]
Thanks @Happypig375 for pointing out the specs. Its definitely helpful.

From

https://github.com/xamarin/Xamarin.Forms/pull/9218

I hope I understand correctly that

"Shapes is exposed using the native APIs on each platform."

**Comment by Happypig375** [comment]
Yes. You can read from https://github.com/xamarin/Xamarin.Forms/pull/9218/files#diff-8cc5e2c00ad0b92522a2ef3076e05e17 onwards.

**Comment by ChaseFlorell** [comment]
Related: https://github.com/dotnet/maui/issues/110

**Comment by WebAssemblyMan** [comment]
Taking a step further. How about Canvas?
